General/intermediate/building-packages/building-packages.md

```diff
@@ -34,5 +34,5 @@ have been built against a more universally available but lower performance
 library. The same holds for optimizing compilers.
 * Curiosity to know more about the tools you use.
 * Pride of building one's tools oneself.
-* For the sheer thrill of building packages. :smile:
```

Comment:
We have something against smileys?

Reply:
@chchang6 apparently Jekyll has something against smileys, it left it as plaintext for the GH pages 😆